plugins/woocommerce-admin/client/task-lists/progress-header/utils.tsx

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-console */
+/**
+ * External dependencies
+ */
+import { Slot, Fill } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { createOrderedChildren, sortFillsByOrder } from '../../utils';
+
+export const WC_TASKLIST_PROGRESS_HEADER_SLOT_NAME =
+	'woocommerce_tasklist_progress_header_item';
```

Comment:
Just learned that we should add `Experimental` somewhere. 😄 

https://github.com/woocommerce/woocommerce/pull/36487#discussion_r1073234410

Reply:
@chihsuan good point! Updated in https://github.com/woocommerce/woocommerce/pull/36482/commits/d2950f87a7be03318a6211e58548673870ceaf0f